### ISSUE TITLE
Direct feedback on Plazi datasets to Plazi

### ISSUE THREAD

**Issue by MattBlissett** [gbif/portal16#1471]
CC @mguidoti, @myrmoteras, @ManonGros 

Plazi have their own community feedback/helpdesk repository at https://github.com/plazi/community

They would like issues regarding Plazi-managed datasets to be created in that repository. We already support directing users towards iNaturalist, PlantNet etc (e.g. click Feedback→Content on https://www.gbif.org/occurrence/3013858297), but we do not create issues in any other GitHub account yet.

**Comment by ManonGros** [comment]
Thanks Matt! It would be best if our system could directly log the feedback on the PLAZI repository: https://github.com/plazi/community

But it isn't possible, I suggest to set up the feedback system to log the issues on GitHub (in our repo) instead of the email. In other words, for the PLAZI datasets, I suggest to have this (without the "We recommend sending a mail to gsautter@gmail.com"):
<img width="500" alt="Screenshot 2021-01-20 at 10 21 48" src="https://user-images.githubusercontent.com/7677271/105154032-61e28300-5b09-11eb-9ecb-30d0bde95445.png">


Instead of this:

<img width="400" alt="Screenshot 2021-01-20 at 10 20 35" src="https://user-images.githubusercontent.com/7677271/105154063-6ad35480-5b09-11eb-9e0b-d46d44eba46b.png">

**Comment by myrmoteras** [comment]
If we get this done, this could also serve as an example on how to curate data in GBIF in near real time. May be installing a list of open issues (date posted) and closed issues (date fixed). This could also work in the opposite way: an institution would want to see open issues on a site on GBIF side.

**Comment by MortenHofft** [comment]
We only try to redirect users on individual occurrences.

Notice that it is already possible to [search for them in github](https://github.com/gbif/portal-feedback/issues?q=is%3Aissue+%22publishingOrgKey%3A+7ce8aef0-9e92-11dc-8738-b8a03c50a862%22+is%3Aopen+). There have been 7 issues related to occurrences from Plazi.

There are other issues related to Plazi of course, but those required a human to notice.

**Redirect to Github for now**
We do not have a system set up to create issues in other organisations repositories (we could look into that). For now we can redirect to the Plazi Github. This of course requires the users to login to leave feedback, but it is useful to know who the user is anyhow.

**Comment by mguidoti** [comment]
Hi @MortenHofft, 

My original suggestion was to create a webhook where I could watch for issues that mention our publishingOrgKey automatically, and then add the issues upon creation to our /plazi/community repo, leaving a note behind to the original creator. In our repo, we do automatically watch and import into our task management system. This would involve granting me some level of access or at least helping me to configure the hook in your repo. Would that be possible?

Despite the automation idea, the suggestion of removing a personal email as our contact to add something more institutional does make sense too.

Best,

**Comment by MortenHofft** [comment]
I do not believe we need to set up any hooks. But I might be mistaken. The Github API allows you to watch the portal-feedback repository, get notifications as well as search issues and leave comments - I think that might be enough to do what you mention?

Before you decide what to do:
* we have 12 open issues mentioning "Plazi". 15 closed. (discarding those created by Donat that I assume you do not want to copy)
* 3 open - 5 closed that mention the publisher UUID.

**Comment by mguidoti** [comment]
Hi @MortenHofft, 

I might be wrong as well. I'll look into this in more detail and see what I can do without any further configuration/access. You might be right on this, which case, I apologize in advance. I'll keep you all posted here.

You are right when you say that I do not want the ones created by Donat, but a filter based in the creator username is totally possible. Plus, this redirect to Plazi Github is great, thanks for it. I loved that after clicking in that button we already have the links to the GBIF record and Plazi's record in the issue body.

Thanks!

![image](https://user-images.githubusercontent.com/44419136/105349754-eba55580-5bc8-11eb-8426-92e41dfc0411.png)

**Comment by mguidoti** [comment]
Hi @MortenHofft,

I couldn't find this repo using one of these third-party apps for integration (Integromat, in this case), using a connection with my personal account (this one). But I can find the ones that I've been either added as a user, or, as a member of the organization. 

If I can be added here or in the GBIF organization I think I'll be able to connect the repos. 

Would that be possible?

**Comment by MortenHofft** [comment]
I've noticed that very few issues are directly related to Plazi, and when they are related, they mostly required helpdesk to diagnose. So I wouldn't myself worry about automating it.

But if you do want to explore that option, then I believe scripting using the Github API should be possible and is closest to what I think you suggest.

Another low-level approach, could be that we start tagging issues with `Plazi`. And you could give the `@gbif/secretariat` a triage role on the Plazi repo? Then we could transfer issues.
